README.md

```diff
@@ -67,7 +67,7 @@ Or the Matrix server (#emulatorjs:matrix.emulatorjs.org):
 <br>
 
 ### Versioning
-There are 3 different version name that you need to be aware of:
+There are three different version names that you need to be aware of.:
```

Comment:
Shouldn't have a period here since its leading into the bullet points (and it looks weird)